### DISCUSSION TITLE
Filtering nucleosomal fragments

### DISCUSSION THREAD

**Discussion by leandro-ventimiglia** [stuart-lab/signac#1813] [General]
Hello,

I started using Signac to analyse my multiome dataset. 
My goal is to find enhancers-promoters interactions. I would like to know if it is a good practice to filter out all the nucleosome fragments (just keeping all the fragments with a length shorter than 140) after the initial clusterization step and before running cicero to find co-accesible peaks.
Thanks

**Comment by timoast** [comment]
This is not something that is typically being done for scATAC-seq analysis
